docs/agent-config.md

```diff
@@ -35,6 +36,24 @@ Here are some quick links into those docs for the configuration options for spec
 * [Span limits](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#span-limits)
 * [Using SPI to further configure the SDK](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#customizing-the-opentelemetry-sdk)
 
+## Supplying configuration properties
+
+Configuration properties can be supplied with one or more of the following ways (priority from highest to lowest):
+* system properties
+* environment variables
+* configuration file
+* SPI
+* hard-coded defaults
+
+### File configuration
+
+You can provide a path to agent configuration file by setting the corresponding property.
+
+| System property                      | Environment variable                 | Description                                                                      |
+|--------------------------------------|--------------------------------------|----------------------------------------------------------------------------------|
+| `otel.javaagent.configuration-file` | `OTEL_JAVAAGENT_CONFIGURATION_FILE` | Path to configuration file which contains a line-oriented list of properties formatted like `property1=value1` or `property2:value2`|
```

Comment:
I think it's fine to just say that it's a Java properties file without describing the format in detail:
```suggestion
| `otel.javaagent.configuration-file` | `OTEL_JAVAAGENT_CONFIGURATION_FILE` | Path to valid Java properties file which contains the javaagent configuration. |
```